packages/debug/src/browser/debug-session-connection.ts

```diff
@@ -243,16 +243,22 @@ export class DebugSessionConnection implements Disposable {
         const connection = await this.connectionPromise;
         const messageStr = JSON.stringify(message);
         if (this.traceOutputChannel) {
-            this.traceOutputChannel.appendLine(`${this.sessionId.substring(0, 8)} theia -> adapter: ${messageStr}`);
+            // keep getMilliseconds
```

Comment:
I believe we can remove the comment, it does not add value (the same goes for `handleMessage`).

Reply:
ok